hooks/600-no-debian.chroot

```diff
@@ -54,4 +54,10 @@ rm -f var/log/bootstrap.log \
 # also remove obsolete cron jobs
 rm -f /etc/cron.daily/dpkg \
       /etc/cron.daily/passwd
-    
+# and services from dpkg
+rm -f var/lib/systemd/deb-systemd-helper-enabled/dpkg-db-backup.timer.dsh-also \
+      var/lib/systemd/deb-systemd-helper-enabled/timers.target.wants/dpkg-db-backup.timer \
```

Comment:
Should we get read of `deb-systemd-helper` and all its files? Should we open an issue about it?

Reply:
Good point, I have changed a bit now the PR to try to remove the remaining dpkg-related files and folders.